scripts/release/prepare-release.sh

```diff
@@ -0,0 +1,278 @@
+    -h, --help        Show this help message
+
+EOF
+}
+
+# Parse command line arguments
+while [[ $# -gt 0 ]]; do
+    case $1 in
+        -d|--dry-run)
+            DRY_RUN=true
+            shift
+            ;;
+        -h|--help)
+            usage
+            exit 0
+            ;;
+        *)
+            log_error "Unknown option: $1"
+            usage
+            exit 1
+            ;;
+    esac
+done
+
+# Main function
+main() {
+    log_info "Starting automated release process..."
+    
+    # Validate repository setup
+    validate_repository
+    
+    # Check if GitHub CLI is installed and authenticated
+    if ! command -v gh > /dev/null 2>&1; then
+        log_error "GitHub CLI (gh) is not installed. Please install it first."
+        exit 1
+    fi
+    
+    # Check if gh is authenticated
+    if ! gh auth status > /dev/null 2>&1; then
+        log_error "GitHub CLI is not authenticated. Please run 'gh auth login' first."
+        exit 1
+    fi
+    
+    # Get current versions
+    log_info "Getting current versions..."
+    read -r current_version_v1 current_version_v2 <<< "$(get_current_versions)"
+    
+    if [[ -z "$current_version_v1" || -z "$current_version_v2" ]]; then
+        log_error "Could not determine current versions"
+        exit 1
+    fi
+    
+    log_info "Current v1 version: $current_version_v1"
+    log_info "Current v2 version: $current_version_v2"
+    
+    # Suggest next versions
+    if [[ "$current_version_v1" =~ ^v([0-9]+)\.([0-9]+)\.([0-9]+)$ ]]; then
+        major_v1="${BASH_REMATCH[1]}"
+        minor_v1="${BASH_REMATCH[2]}"
+        patch_v1="${BASH_REMATCH[3]}"
+        suggested_v1="v$major_v1.$((minor_v1 + 1)).0"
+    else
+        log_error "Could not parse v1 version: $current_version_v1"
+        exit 1
+    fi
+    
+    if [[ "$current_version_v2" =~ ^v([0-9]+)\.([0-9]+)\.([0-9]+)$ ]]; then
+        major_v2="${BASH_REMATCH[1]}"
+        minor_v2="${BASH_REMATCH[2]}"
+        patch_v2="${BASH_REMATCH[3]}"
+        suggested_v2="v$major_v2.$((minor_v2 + 1)).0"
+    else
+        log_error "Could not parse v2 version: $current_version_v2"
+        exit 1
+    fi
+    
+    # Get user input for new versions
+    read -p "New v1 version [$suggested_v1]: " user_version_v1
+    if [[ -z "$user_version_v1" ]]; then
+        user_version_v1="$suggested_v1"
+    fi
+    if [[ ! "$user_version_v1" =~ ^v[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+        user_version_v1="v$user_version_v1"
+    fi
+    
+    read -p "New v2 version [$suggested_v2]: " user_version_v2
+    if [[ -z "$user_version_v2" ]]; then
+        user_version_v2="$suggested_v2"
+    fi
+    if [[ ! "$user_version_v2" =~ ^v[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+        user_version_v2="v$user_version_v2"
+    fi
+    
+    new_version_v1="$user_version_v1"
+    new_version_v2="$user_version_v2"
+    
+    log_success "Using new versions: v1=$new_version_v1, v2=$new_version_v2"
+    
```

Comment:
the versions should be given as input and determined by start.sh, no need to re-derive them here. See description in #7500. The only thing we may want to do here is validate that the versions are in the right format.
```suggestion
    # Get current versions
```
```suggestion
    
```